### PR TITLE
Revise the context

### PR DESCRIPTION
**Pull request by Spin0530**
Revise the context

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -178,8 +178,9 @@
 				<div class="text-inner">
 					<div class="row">
 						<div class="col-md-8 col-md-offset-2 text-center">
-							<h1 class="to-animate">Enable your business with <span>Blockchain</span></h1>
-							<h2 class="to-animate">Build the trusted economy in IoT, securing data with IOTA <a href="http://creativecommons.org/licenses/by/3.0/" target="_blank"> </a>
+							<h1 class="to-animate">Start your business with <span>Blockchain Technology</span></h1>
+							<h2 class="to-animate">Build up the reliable IoT system, Securing Data with IOTA
+								<a href="http://creativecommons.org/licenses/by/3.0/" target="_blank"> </a>
```
Comment (jserv) [inline docs/index.html]: Why would you insert CC link here?

File: docs/index.html [hunk 2]
```diff
@@ -203,10 +204,10 @@ <h2 class="to-animate">Build the trusted economy in IoT, securing data with IOTA
 			<div class="container">
 				<div class="row">
 					<div class="col-md-12 section-heading text-center">
-						<h2 class="to-animate"><span>We provide</span></h2>
+						<h2 class="to-animate"><span>Our Service</span></h2>
 						<div class="row">
 							<div class="col-md-8 col-md-offset-2 subtext">
-								<h3 class="to-animate">Facing to the upcoming Machine Economy, over 5 Trilion transactions and microtrasactions happens everday by 2020, BiiLabs works with <a href="https://www.iota.org/">IOTA Foundation</a> closely for next revolution (Internet of Trusted) in each industry. </h3>
+								<h3 class="to-animate">Facing with the ongoing “Machine Economics,” over 5 Trillion transactions and microtransactions taking place daily by 2020, BiiLabs intends to provide distributed technology as next-generation leverage for each industry, in cooperation with IOTA Foundation. </h3>
```
Comment (jserv) [inline docs/index.html]: Why "transactions and microtransactions" here?

Comment (jserv) [reply docs/index.html]: Change "distributed technology" to "distributed ledger infrastructure".

File: docs/index.html [hunk 3]
```diff
@@ -241,7 +242,7 @@ <h3>Accelerate </h3>
 							</div>
 							<div class="fh5co-post to-animate">
 								<h3>Customized API</h3>
-								<p>BiiLabs APIs based on IOTA Tangle enable your service like Digital ID, ledger & settlement, digital forensics, micro-payment, etc. on blockchain. We provide and secure sandbox for testing and make your service on blockchain would be super easy.</p>
+								<p>Based on IOTA Tangle, BiiLabs realizes the service such as Digital ID, ledger & settlement, micro-payment, digital forensics, etc. by both providing and securing sandbox for testing, which means the customization of API.</p>
```
Comment (jserv) [inline docs/index.html]: Is it appropriate to use "realize"?

File: docs/index.html [hunk 4]
```diff
@@ -282,8 +283,8 @@ <h3>TangleID </h3>
 								<span><i class="icon-globe"></i></span>
 							</div>
 							<div class="fh5co-post to-animate">
-								<h3>Scalable Data Business</h3>
-								<p>For the first time ever true micro and even nano-transactions are enabled due to the unique IOTA Tangle architecture, providing developers with a brand new set of tools for their applications in both IoT and Web.</p>
+								<h3>Scalability in Business</h3>
```
Comment (jserv) [inline docs/index.html:286]: "Scalability in Business" is not always meant to be scalable business, right?

File: docs/index.html (hunk 3 above)
Comment (Spin0530) [reply docs/index.html]: perhaps "implement the service" would be better

File: docs/index.html (hunk 2 above)
Comment (Spin0530) [reply docs/index.html]: according to previous version, I suppose this usage means to emphasize on the realization of micro-payment records

File: docs/index.html (hunk 1 above)
Comment (Spin0530) [reply docs/index.html]: I'll check it out to remove the unnecessary link

File: docs/index.html (hunk 4 above)
Comment (Spin0530) [reply docs/index.html:286]: Not sure how to correct it, but I find it more wide-covered in that usage.

**Pull request merged**